_component_design/status-labels.md

```diff
@@ -76,13 +76,13 @@ Status labels differentiate themselves from buttons and links by utilizing 100%
 
 ### Emphasis
 
-The primary UX goal of the status label component is to emphasize or de-emphasize data within a UI. This is performed via the following properties: stroke, fill, and color.
+The primary UX goal of the status label component is to emphasize or de-emphasize data within a UI. This is performed by modifying the stroke and fill or color property on a status label.
```

Comment:
The status label is used to communicate the status of an object or process. You can emphasize or de-emphasize that status data  by modifying the stroke and fill or color property on a status label.

Reply:
I would remove the closed potion, because I'm not sure what 'closed' is referring to here. I could see an implementation where status labels are asynchronously updated or added to an object that is being displayed.

Reply:
ok remove that 2nd sentence.

Reply:
updated.